src/ini2toml/types.py

```diff
@@ -17,7 +17,7 @@
 if sys.version_info <= (3, 8):  # pragma: no cover
     # TODO: Import directly when `python_requires = >= 3.8`
     if TYPE_CHECKING:
-        from typing_extensions import Protocol
```

Comment:
What is the minimum Python supported?  Pyupgrade made this change because I said `--py38-plus`

Reply:
Hi @cclauss, I am committed to follow Python's EoL. So this would mean supporting Python 3.7 until 2023-06-27.